### PR TITLE
Remove isomorphic-webcrypto dependency

### LINKED ISSUE #62: Bundling with Parcel fails
When bundling within a Parcel/typescript project, the isomorphic-crypto import is actually under `crypto.default`, not the top-level `crypto` import, so references to it return undefined. I expect this is because it treats the dependency as a [modern es6 module](https://github.com/kevlened/isomorphic-webcrypto/blob/master/package.json#L6) rather than a commonjs require (or something like that, i don't really understand JS anyway).

Anyway, importing the `urbit-key-generation/dist` is a fine stopgap for now.

### PR DESCRIPTION
**Pull request by jtobin**
(Resolves #62)

Uses `js-sha256`, a more popular and seemingly more suitable library, instead.

### CODE REVIEW

File: test/assets/wallet0.json [hunk 1]
```diff
@@ -1 +1 @@
...
...348248b318859813ce3830a221d86","private":"74cbbc06a4339ce8b72ed7e84597d6eb6b2e3d90e672dded5048b2000ed9e33e","chain":"63740b42257aeb76ef539fd208e57bc0d15f2d34b27a99729fab2025a9771387","address":"0xd0f7D56290675609d37FC75F8413535f7aa5DB04"}},"voting":{"seed":"noise vote enter doctor double charge above car tomato brick dutch miracle narrow seed aware harsh bottom armor bonus property census hybrid target symbol","keys":{"public":"028df25e7fabe0b2c74d9d65dcc8007598a560f863498c98b490da420a58e115c1","private":"479fd82a0695563d29b3942bf180f4614a6736f1b731b9df33894bb66a79ca28","chain":"04f0d00f571a833ebd62576424fd084194c3e1488c162bb979227a03fdc38d16","address":"0x38A9e280124b3bA1eaac446643b60752A0663769"}},"management":{"seed":"veteran flash code abuse course front better grace fabric chef puzzle outside visual sauce spice auction cactus galaxy need view tomorrow spirit chimney course","keys":{"public":"03bc9499271df27b4e90cd13a63ba6f48b88aaaca75ed9a7ed97429ef9d7bb4dea","private":"5bed7cc50cb74cd80a798bdb2bc4c7effb7d6875a60d0d0d6ec81efd90fce724","chain":"1135c11e847896630c8bd3c356097f937b6df92d88e2c239f517fa0653ab205a","address":"0x0177441D1b78FaEEEe95c217beBDF02DEB2ce5ba"}},"network":{}}
```
Comment (Fang-) [inline test/assets/wallet0.json]: Can we maybe stop caring about version numbers for the equality checks on these? Seeing these files change when no crypto-level changes have been made feels not good.

File: src/index.js [hunk 2]
```diff
@@ -120,11 +120,12 @@ const argon2u = async (entropy, ship) => {
  * SHA-256 hash function.
  *
  * @param  {Array, ArrayBuffer, Buffer, String} args any number of arguments
- * @return {Promise<ArrayBuffer>}  the hash, as bytes
+ * @return {<Buffer>}  the hash, as bytes
```
Comment (Fang-) [inline src/index.js]: Surely there's no weird js stuff that can go wrong with this. (^:

File: test/assets/wallet0.json (hunk 1 above)
Comment (jtobin) [reply test/assets/wallet0.json]: Yeah that'd be great, just requires a suitable implementation in the test suite.

File: src/index.js (hunk 2 above)
Comment (jtobin) [reply src/index.js]: Haha.  I think it was fine, but I took another pass over everything just to be sure.. 😄 

The functions that use `sha256` (and the function that use them, etc.) can be de-async'd, so I've done that.  Also there was one superfluous use of `Buffer.from` so best nix that as well.

Comment (Fang-) [reply src/index.js]: Thanks! This still looks good to me. (:

**Pull request merged**